From a18c0b5955a2f6281bc89d1add3f24907813ae31 Mon Sep 17 00:00:00 2001
From: Hanumanth Reddy Pothula <c_hpothu@codeaurora.org>
Date: Thu, 23 Nov 2017 12:24:33 +0530
Subject: qcacld-2.0: Resolve compilation error

Resolve compilation error caused by change-id,
Ida655f83630c369df746e7c0c9d61a8fee2932a2.

Change-Id: Ied6328ce2ed1e49dc56697ddead4d3abb6e62875
CRs-Fixed: 2120605
---
 CORE/SERVICES/WMA/wma.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CORE/SERVICES/WMA/wma.c b/CORE/SERVICES/WMA/wma.c
index 852cc72..651e9be 100644
--- a/CORE/SERVICES/WMA/wma.c
+++ b/CORE/SERVICES/WMA/wma.c
@@ -5185,7 +5185,6 @@ static int wma_unified_link_radio_stats_event_handler(void *handle,
 
 #endif /* WLAN_FEATURE_LINK_LAYER_STATS */
 
-/**
 u_int8_t *wma_add_p2p_ie(u_int8_t *frm)
 {
 	u_int8_t wfa_oui[3] = WMA_P2P_WFA_OUI;
-- 
cgit v1.1

